time imgtime
float64 num
